__tests__/app.test.js

```diff
@@ -113,3 +113,91 @@ describe("CORE: GET /api/articles", () => {
       });
   });
 });
+
+describe("CORE: GET /api/articles/:article_id/comments", () => {
+  test("GET: 200, should get all comments for an article", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then((response) => {
+        const { comments } = response.body;
+        comments.forEach((comment) => {
+          expect(comment.article_id).toBe(1);
```

Comment:
We should also be checking for article with id 1, that we are getting the correct number of articles back. We should also be checking for the existence of the following properties: 
comment_id
votes
created_at
author 
body